parser/parse.go

```diff
@@ -458,6 +458,7 @@ func (b *Builder) findTypesIn(pkgPath importPathString, u *types.Universe) error
 	for _, f := range b.parsed[pkgPath] {
 		if strings.HasSuffix(f.name, "/doc.go") {
 			tp := u.Package(string(pkgPath))
+			tp.Comments = []string{}
```

Comment:
Please include a // TODO explaining this line since if neither thockin nor wojtek-t figured out what was here for it must be pretty mysterious.

Then I'll hold my nose and lgtm.

I feel like taking a week off just to fix all the things that have been done to this library :(